src/darwin/Framework/CHIP/MTRCommissionableBrowserDelegate.h

```diff
@@ -26,12 +27,12 @@ MTR_NEWLY_AVAILABLE
 /**
  * Tells the delegate the commissionable manager discovered a device while scanning for devices.
  */
-- (void)didDiscoverCommissionable:(MTRCommissionableBrowserResult *)device;
+- (void)didDiscoverCommissionable:(MTRDeviceController *)controller device:(MTRCommissionableBrowserResult *)device;
```

Comment:
I think the naming on this should be more like:
```suggestion
- (void)controller:(MTRDeviceController *)controller discoveredCommissionableDevice:(MTRCommissionableBrowserResult *)device;
```
or so.

Reply:
I end up naming those `did[Find|Remove]CommissionableDevice`. I have taken inspiration from https://developer.apple.com/documentation/foundation/nsnetservicebrowserdelegate?language=objc